test/e2e/utils/common.go

```diff
@@ -30,7 +31,25 @@ const (
 	truststoreSecretSuffix = "truststore"
 )
 
-var VersionManager = func() *version.Manager {
+// var VersionManager = func() *version.Manager {
+func VersionManager(testKube *TestKubernetes) *version.Manager {
+	// If Operator is running via OLM, fetch the envs
+	if RunLocalOperator == "FALSE" {
```

Comment:
This won't work if we want to run the tests against an Operator deployed via `skaffold run`.

Reply:
So should we change the env to be rather enum then true false? Is skaffold something we are using at the moment?

Reply:
> So should we change the env to be rather enum then true false?

Tbh I'm not sure this should exist upstream. We already have the `INFINISPAN_OPERAND_VERSIONS` env variable that allows you to set the versions used by the tests. Can't you execute the equivalent of the `SetRelatedImagesEnv` in a script on Jenkins before executing the downstream testsuite?

> Is skaffold something we are using at the moment?

Yes, it greatly simplifies deploying and remote debugging the operator in development. Basic instructions are on the README.